src/Entity/DfCategoryBuild.php

```diff
@@ -47,7 +47,9 @@ public function build($json = true)
         $this->assign();
 
         foreach ($this->categories as $category) {
-            $payload[] = $this->buildCategory($category);
+            if (\Category::categoryExists($category)) {
+                $payload[] = $this->buildCategory($category);
+            }
```

Comment:
Another way (but your code is excellent too, just to comment it). Guard clause way:

```suggestion
            if (!\Category::categoryExists($category)) {
                continue;
            }
            $payload[] = $this->buildCategory($category);
```

Reply:
great!